x/stakeibc/keeper/posthandler.go

```diff
@@ -0,0 +1,76 @@
+package keeper
+
+import (
+	"fmt"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/Stride-Labs/stride/v11/x/stakeibc/types"
+)
+
+type StakeIbcPostDecorator struct {
+	StakeIbcKeeper Keeper
+}
+
+func NewStakeIbcPostDecorator(stakeIbcDecorator Keeper) StakeIbcPostDecorator {
```

Comment:
Will this check only run after transactions in the stakeibc module? I think ideally, it should run after every transaction

Reply:
It runs after all txs. I passed stakeibckeeper for using some funcs